src/components/property/property.jsx

```diff
@@ -0,0 +1,293 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {PlaceType} from '../../const';
+
+const Property = (props) => {
+  const {property} = props;
+  const {type, description, rating, price, isPremium, isFavourite, photoSet, bedrooms, adultsMax, goods, host} = property;
```

Comment:
декомпозицию можно делать сразу в аргументах

```suggestion
const Property = ({ 
  property: {
    type, description, rating, price, isPremium, isFavourite, 
    photoSet, bedrooms, adultsMax, goods, host
  } 
}) => {
```

Reply:
ого. не знал. возьму на вооружение =)